air_quality_management/myapp/views.py

```diff
@@ -133,7 +133,12 @@ def air_quality_data_detail(request, data_id):
 
 @login_required
 def profile(request):
-    return render(request, 'user/user_profile.html')
+    air_quality_data = AirQualityData.objects.filter(provider=request.user.username)[:2]
```

Comment:
sao chỗ này chỉ lấy 2 bản ghi e nhể

Reply:
Chỗ này em để hiển thị ngắn gọn 2 report gần nhất trên trang profile nên em lấy 2 bản ghi thôi ạ. Còn khi vào toàn bộ lịch sử thì em mới để toàn bộ bản ghi.